NitroxServer/Server.cs

```diff
@@ -57,6 +57,7 @@ public string SaveSummary
                 builder.AppendLine($" - Encyclopedia entries: {world.GameData.PDAState.EncyclopediaEntries.Count}");
                 builder.AppendLine($" - Storage slot items: {world.InventoryManager.GetAllStorageSlotItems().Count}");
                 builder.AppendLine($" - Inventory items: {world.InventoryManager.GetAllInventoryItems().Count}");
+                builder.AppendLine($" - Partially unlocked: {world.GameData.PDAState.PartiallyUnlockedByTechType.Count}");
```

Comment:
As it's has no "Tech" in the message there is no clear understanding what is really partial unlocked.